MultiTable/src/RealtimeChangeHandler.js

```diff
@@ -17,7 +17,7 @@ const INVALID_ARG = 'MultiTable|RealtimeChangeHandler.handle|Invalid arg: ';
 
 function onSampleAdd(data, sample) {
   // console.log(new Date(), 'onSampleAdd', sample);
-  if (SampleUtils.isUnderRootSubject(sample, data.rootSubject)) {
+  if (data.rootSubject && SampleUtils.isUnderRootSubject(sample, data.rootSubject)) {
```

Comment:
Would it be "cleaner" to modify SampleUtils.isUnderRootSubject( to just return false if the second arg is falsey, rather than modifying all of the callers of that fn?  e.g.

```
  static isUnderRootSubject(sample, rootSubject) {
    return rootSubject && rootSubject.length > 0 && sample && sample.name && sample.name.toLowerCase()
      .startsWith(rootSubject.toLowerCase());
  } // isUnderRootSubject
```

Reply:
Used an if condition instead to clearly return true/false instead of null/undefined in some cases when one of the condition is null/undefined.